tests/test_confidence.py

```diff
@@ -0,0 +1,89 @@
+from vulture import core
+
+from . import skip_if_python_2
+from . import skip_if_python_3
+
+dc = core.DEFAULT_CONFIDENCE
+
+
+def check_min_confidence(code, min_conf, tests):
+    v = core.Vulture(verbose=True, min_confidence=min_conf)
+    v.scan(code)
+    for item, (lineno, confidence) in zip(v.get_unused_code(), tests):
+        assert item.lineno == lineno
```

Comment:
Instead of the lineno, I think we should compare item.name. Can you adapt the calls, please?

```
def check_min_confidence(code, min_confidence, **expected):
    v = core.Vulture(verbose=True, min_confidence=min_confidence)
    v.scan(code)
    detected = dict((item.name, item.confidence) for item in v.get_unused_code())
    assert detected == expected

# Example call:
check_min_confidence(code, 50, foo=90)
```